compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/diagnostics/DiagnosticErrorCode.java

```diff
@@ -239,7 +240,7 @@
     // Expression followed by the start keyword must be a func-call, a method-call or a remote-method-call
     ERROR_INVALID_EXPRESSION_IN_START_ACTION("BCE207", "error.invalid.expression.in.start.action"),
     // Cannot have the  same qualifier twice
-    ERROR_SAME_OBJECT_TYPE_QUALIFIER("BCE208", "error.same.object.type.qualifier"),
+    ERROR_SAME_CLASS_TYPE_QUALIFIER("BCE208", "error.same.class.type.qualifier"),
```

Comment:
Should we rename this to ERROR_DUPLICATE_CLASS_TYPE_QUALIFIER? it seems consistent with the rest.